markhor_flippers/launch/flippers.launch

```diff
@@ -1,5 +1,5 @@
 <launch>
-    <arg name="robot_namespace" default="/markhor" />
+    <arg name="robot_namespace" default="/markhor/flippers" />
```

Comment:
Est-ce que c'est sa qui fix les flippers qui launchaient pas du premier coup?

Reply:
Yes, this segment the whole flippers nodes inside their own namespace and avoid node collision which would kill the node.